### ISSUE TITLE
In Elixir registered names of a process should not have the 'Elixir.' prefix

### ISSUE THREAD

**Issue by aboroska** [andytill/erlyberly#142]
See screenshot where it is visible: https://twitter.com/arathunku/status/826852033584054273

**Comment by andytill** [comment]
@aboroska is this the issue you fixed in the last PR?

**Comment by aboroska** [comment]
No, see Reg.Name in this screenshot. It should be `AMQP_POOL` without Elixir prefix and apostrophes. Alternatively keep the current Erlang style and close the ticket as perhaps showing processes in Erlang atom style is more beneficial. Also note that it is probably not a trivial fix.

<img width="698" alt="screen shot 2017-04-11 at 16 15 40" src="https://cloud.githubusercontent.com/assets/1858479/24913739/5bae52ba-1ed2-11e7-9237-038315301f7c.png">

**Comment by andytill** [comment]
Thanks, just me getting confused.
